packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts

```diff
@@ -900,8 +907,8 @@ async function askOpenIDConnectQuestions(authSettings) {
   };
 }
 
-function validateDays(input) {
-  const isValid = /^\d+$/.test(input);
+async function validateDays(input) {
```

Comment:
does this need to be async?

Reply:
Doesn't have to be async. But inquirer can handle validate method returning promise, so better to use it wherever possible.